rexrov2_description/package.xml

```diff
@@ -1,16 +1,19 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>rexrov2_description</name>
-  <version>0.1.3</version>
+  <version>0.1.0</version>
   <description>The robot description files for the RexROV 2 underwater vehicle and launch files to spawn the model in different configurations.</description>
 
   <maintainer email="Musa.Marcusso@de.bosch.com">Musa Morena Marcusso Manhaes</maintainer>
   <maintainer email="sebastian.scherer2@de.bosch.com">Sebastian Scherer</maintainer>
   <maintainer email="LuizRicardo.Douat@de.bosch.com">Luiz Ricardo Douat</maintainer>
+  <maintainer email="borgesjvt@gmail.com">João Victor Torres Borges</maintainer>
 
   <author email="Musa.Marcusso@de.bosch.com">Musa Morena Marcusso Manhaes</author>
   <author email="sebastian.scherer2@de.bosch.com">Sebastian Scherer</author>
   <author email="LuizRicardo.Douat@de.bosch.com">Luiz Ricardo Douat</author>
+  <author email="borgesjvt@gmail.com">João Victor Torres Borges</author>
```

Comment:
On the other hand, I think we should not add ourselves as authors until we make more significant updates

Reply:
I think it i safe to remove them and only keep a note on our copyright file since this is a different package